application/features/bootstrap/SyncContext.php

```diff
@@ -195,22 +196,21 @@ public function iSyncAllTheUsersFromTheIdStoreToTheIdBroker()
      */
     public function onlyTheFollowingUsersShouldExistInTheIdBroker(TableNode $table)
     {
+        foreach ($table as $row) {
```

Comment:
I like the way your suggestion looks, but for whatever reason, it doesn't return the same data.